src/connectors/dispatcher/impl.js

```diff
@@ -50,8 +49,7 @@ module.exports = new class extends Connector {
     }
 
     async fetchPlaybookRuns (filter = null, fields = null, sort_by = null) {
-        const uri = new URI(config.platformHostname);
-        uri.path('/api/playbook-dispatcher/v1/runs');
+        const uri = this.buildUri(host, 'playbook-dispatcher', 'v1', 'runs');
```

Comment:
I think you'll need a `/api` in the uri

Reply:
Ah!  Yup!  adding it now!

Reply:
@Jason-RH I stand corrected!  We actually add the /api prefex automatically so manually adding it here is not necessary.

Reply:
Ah, my bad!